Scripts/parameters/zone.py

```diff
@@ -93,6 +93,12 @@
         "area": "all",
     },
 )
+purpose_areas = {
+    "metropolitan": (0, 6000, 16000),
+    "peripheral": (16000, 31000),
+    "all": (0, 6000, 31000),
+    "external": (31031, 40000),
+}
```

Comment:
Consider adding a helpful comment here because it is difficult to understand why some areas have three items and some only two.